corehq/tests/noseplugins/elasticsnitch.py

```diff
@@ -1,69 +0,0 @@
-"""A test timing plugin for nose
-
-Usage: ./manage.py test --with-elasticsnitch --snitch-out=/path/to/elasticsnitch.txt
-"""
-import sys
-
-from nose.plugins import Plugin
-from corehq.tests.noseplugins.uniformresult import uniform_description
-from corehq.apps.es.client import manager
-
-
-class ElasticSnitchPlugin(Plugin):
```

Comment:
@millerdev curious...but is something like this still useful/needed? Do we have a backlog to track re-implementing some of these plugins with `pytest`? I know the commit message says out of scope, but this one in particular seems like it was added fairly recently with the ES migration work (2023) by Joel.

Reply:
My thought was we can reimplement this if/when we need it. I doubt anyone has used it since Joel finished his ES work, but I'd be happy to port it to pytest if someone wants it. Do you use it?